docs/integrations/bullmq.md

```diff
@@ -14,22 +14,24 @@ By replacing Redis with Dragonfly, you can achieve superior performance and scal
 
 ## Running BullMQ with Dragonfly
```

Comment:
I would even add a `## TL;DR` section above this one, saying something like: if you can set add hash-tags to your queue name or prefix (e.g. `{queue1}` instead of `queue1`), use `./dragonfly ...`
wdyt?

Reply:
Added! 😃